### DISCUSSION TITLE
Is there a video for everything beyond chapter 4.

### DISCUSSION THREAD

**Discussion by JonasDennisHansen** [mrdbourke/pytorch-deep-learning#691] [General]
The book version of the course has been updated to have 9 chapters now, but in the 25 hour video Daniel only covers up until chapter 4. Has he made a new video somewhere covering all the new chapters or do you know if its in process?

Thanks in advance!

**Comment by mrdbourke** [comment]
Hey @milehighprofit , 

You can find all the videos for the rest of the course (another 150+) on Zero to Mastery - https://dbourke.link/ZTMPyTorch

Or on Udemy.

**Comment by shisirkha** [comment]
Yeah, if you want to support the creator of the course you can join their paid course here `https://dbourke.link/ZTMPyTorch`.

Btw Always remember if you want to learn more, you need to just see the documentation for the specific problem you are trying to learn how to solve like chatbots that uses RNNs (Recurrent neural network).

basically, these types of neural networks are designed to predict the next word by just using the output as input again and again until there nothing left to do.

Good luck learning.

oh btw see these for your specific problems:

Linear regression and classification problems : PyTorch - https://pytorch.org/docs
computer vision problems: torchvision - https://pytorch.org/vision
Audio specific problems: torchaudio - https://pytorch.org/audio
text specific problems(also chatbots are text problems): torchtext - https://pytorch.org/text
models publish problems: torchserve - https://pytorch.org/serve/


There are more then these.
